.buildkite/scripts/run_models.sh

```diff
@@ -17,5 +17,5 @@ echo `pwd`
 cd integration_tests
 dbt deps
 dbt seed --target "$db" --full-refresh
-dbt run --target "$db" --full-refresh --vars '{ad_reporting__linkedin_ads_enabled: false, ad_reporting__pinterest_ads_enabled: false, ad_reporting__tiktok_ads_enabled: false, ad_reporting__snapchat_ads_enabled: false, ad_reporting__apple_search_ads_enabled: false}' 
-dbt test --target "$db" --vars '{ad_reporting__linkedin_ads_enabled: false, ad_reporting__pinterest_ads_enabled: false, ad_reporting__tiktok_ads_enabled: false, ad_reporting__snapchat_ads_enabled: false, ad_reporting__apple_search_ads_enabled: false}' 
+dbt run --target "$db" --full-refresh --vars '{ad_reporting__linkedin_ads_enabled: false, ad_reporting__pinterest_ads_enabled: false, ad_reporting__tiktok_ads_enabled: false, ad_reporting__snapchat_ads_enabled: false, ad_reporting__apple_search_ads_enabled: false, ad_reporting__amazon_ads_enabled: false}' 
+dbt test --target "$db" --vars '{ad_reporting__linkedin_ads_enabled: false, ad_reporting__pinterest_ads_enabled: false, ad_reporting__tiktok_ads_enabled: false, ad_reporting__snapchat_ads_enabled: false, ad_reporting__apple_search_ads_enabled: false, ad_reporting__amazon_ads_enabled: false}' 
```

Comment:
These are actually disabling the stated package models from running in the BuildKite job.

I am curious to get your thoughts on if we should omit amazon ads here, or instead swap one of the packages that is not being edited to run? I would love to get @fivetran-sheringuyen input here as well.

So my suggestion is the following:
- Amazon Ads is being added, so we do in fact want to test the changes
- Therefore, we do not omit it from the tests and rather omit Facebook Ads as it has had no changes in this latest update
- Then if we update TikTok Ads next, we do not omit it from the tests
- Rather we omit Amazon Ads as it has not had any updates.

What are your thoughts?

Reply:
Done. Swapped to omit Facebook Ads.